pipe_events/fishing_events_incremental.py

```diff
@@ -74,14 +74,19 @@ def run(bq, params):
 
     bq.end_session(session_id)  # required to use destination in QueryJobConfig then
 
-    log.info("*** 5. Runs the filter over the merged table.")
+    log.info("*** 5. Runs the filter over the merged table with truncated data.")
     filter_query = bq.format_query("fishing-events-3-filter.sql.j2", **params_copy)
     bq.run_query(
         filter_query,
         dest_table=params_copy["filtered_table"],
+        write_disposition="WRITE_TRUNCATE",
         partition_field="event_end_date",
         clustering_fields=["event_end_date", "seg_id"],
         labels=params["labels"],
-    )  # with session cannot set destination
+    )  # Ends the session previously because cannot set the destination with it
+    bq.update_table_schema(
+        params_copy["filtered_table"],
+        "./assets/bigquery/fishing-events-3-filter-schema.json"
+    )  # schema should be kept after trucate
```

Comment:
Why do we need to update table schema at this point? is it possible that running the query above will remove it?